### PR TITLE
Check the lag with pgroonga_wal_status()

### PR DESCRIPTION
**Pull request by abetomo**
* List PGroonga indexes with unapplied WAL in pgroonga_wal_status()
* List all PGroonga indexes if primary log position and lag

### CODE REVIEW

**Comment by abetomo** [comment]
If this looks good, revert as https://github.com/pgroonga/pgroonga/commit/c2a01c0d1ee587ee063aede3510d9a9adcbb1b43 is not needed.

File: data/pgroonga.sql [hunk 1]
```diff
@@ -435,9 +435,18 @@ CREATE FUNCTION pgroonga_wal_set_applied_position()
...
-	AS 'MODULE_PATHNAME', 'pgroonga_list_lagged_indexes'
-	LANGUAGE C
+	RETURNS TABLE (name text) AS $$
+		SELECT name FROM pgroonga_wal_status()
+		WHERE current_block != last_block
+			OR current_offset != current_offset
+			OR current_size != last_size
+		UNION
+		SELECT name FROM pgroonga_wal_status()
+		WHERE EXISTS(
```
Comment (kou) [inline data/pgroonga.sql:445]: Can we use one `SELECT`?

```suggestion
		OR EXISTS(
```

Comment (abetomo) [reply data/pgroonga.sql]: Good point.

**Comment by kou** [comment]
We can remove the C implementation.

File: data/pgroonga.sql [hunk 2]
```diff
@@ -434,12 +434,45 @@ CREATE FUNCTION pgroonga_wal_set_applied_position()
...
+					OR current_offset != current_offset
+					OR current_size != last_size
+					OR EXISTS(
+						SELECT 1 FROM pg_stat_wal_receiver
+						WHERE flushed_lsn != latest_end_lsn
+					);
+			' LANGUAGE SQL
+			STRICT
+			PARALLEL SAFE;
+	ELSIF SUBSTR(rec.setting, 1, 2) = '12' THEN
```
Comment (kou) [inline data/pgroonga.sql:459]: We can use `ELSE` because we don't support PostgreSQL 11 or earlier.

**Pull request merged**